### PR TITLE
Fix compile failure issue when docs/_build is present

### PR DESCRIPTION
**Pull request by snleee**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1140,6 +1143,9 @@
             <!-- Sphynx auto generated makefile -->
             <exclude>docs/Makefile</exclude>
 
+            <!-- Exclude build files for docs -->
+            <exclude>docs/_build/**</exclude>
```
Comment (Jackie-Jiang) [inline pom.xml]: Should we ignore .rst files in rat?

Comment (snleee) [reply pom.xml]: From some of the voting mails that i checked, they do check if documentation files (.md or .rst) have the proper headers. So, I would prefer to keep them from the rat check.

Comment (Jackie-Jiang) [reply pom.xml]: But we get the following warning when we build the project:

Comment (snleee) [reply pom.xml]: Got it. For now, I removed the check from license-maven-plugin. But, during the release process, RAT plugin will check (RAT checks the valid license but not exact text given in `LICENSE` file.) 

`mycila/license-maven-plugin` does not natively support rst file nor support `..` comment style. http://code.mycila.com/license-maven-plugin/

**Pull request merged**